src/main/java/lottogame/LottoMain.java

```diff
@@ -13,12 +13,20 @@ public class LottoMain {
 
     public static void main(String[] args) {
         Money money = InputView.requestMoney();
+        ResultView.printLinkBreak();
         LottoGame lottoGame = new LottoGame();
-        List<Lottos> lottos = lottoGame.createLottos(money);
-        ResultView.printCountOfPurchaseLottos(lottos);
+        Number countOfLottos = lottoGame.calculateCountOfLottos(money);
+        Number manualCountOfLotto = InputView.requestCountOfManualLottos(countOfLottos);
+        ResultView.printLinkBreak();
+        List<List<Number>> manualInputNumbers = InputView.requestMultipleManualLottos(manualCountOfLotto);
+        List<Lottos> multipleManualLottos = lottoGame.createMultipleLottos(manualInputNumbers);
+        List<Lottos> multipleAutoLottos = lottoGame.createMultipleLottos(countOfLottos.minus(manualCountOfLotto));
+
+        List<Lottos> lottos = lottoGame.merge(multipleManualLottos, multipleAutoLottos);
+        ResultView.printAutoAndManualLottosCount(multipleManualLottos, multipleAutoLottos);
         ResultView.printLottos(lottos);
```

Comment:
```suggestion
        LottoBundle lottoBundle = LottoBundle.manual(lottoGame.createMultipleLottos(manualInputNumbers));
        lottoBundle.auto(lottoGame.createMultipleLottos(countOfLottos.minus(manualCountOfLotto)));

        ResultView.printAutoAndManualLottosCount(lottoBundle.toAutoLottos(), lottoBundle.toManualLottos());
        ResultView.printLottos(lottoBundle.all());
```

먼가 일반적인 자료구로 많은 부분이 구성되어있어서 약간의 리팩토링을 해보았는데요.
위는 간략하게 리팩토링 해본것이고 더 좋은 방법은 분명 있을꺼라고 생각합니다!

말씀드리고 싶은부분은 좀 더 객체간의 대화를 통해 구성해보는것이 어떨가 싶어서요!

Reply:
[1d44c9d](https://github.com/next-step/java-lotto/pull/3753/commits/1d44c9df2bc52741169279ce13f6270a119fd3c5) 에 적용했습니다!
`LottosBundle` 을 도입하니깐 `lottoGame`에 맞지 않는 책임이 많이 덜어 졌습니다 😄